src/features/chat-message/controllers/chat-message.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { Controller, Get, Param, Query } from '@nestjs/common';
+import { routesV1 } from '@src/configs/app.route';
+import { QueryBus } from '@nestjs/cqrs';
+import { ApiTags } from '@nestjs/swagger';
+import { SetPagination } from '@src/libs/interceptors/pagination/decorators/pagination-interceptor.decorator';
+import { FindChatMessagesQuery } from '@src/features/chat-message/queries/find-chat-messages/find-chat-messages.query';
+import { Paginated } from '@src/libs/types/type';
+import { ChatMessageEntity } from '@features/chat-message/domain/chat-message.entity';
+import { ChatMessageMapper } from '@src/features/chat-message/mappers/chat-message.mapper';
+import { ApiInternalServerErrorBuilder } from '@src/libs/api/decorators/api-internal-server-error-builder.decorator';
+import { FindChatMessagesRequestQueryDto } from '@features/chat-message/dtos/request/find-chat-messages.request-query-dto';
+import { ChatMessageResponseDto } from '@features/chat-message/dtos/response/chat-message.response-dto';
+import { ApiChatMessage } from '@features/chat-message/controllers/chat-message.swagger';
+import { ParsePositiveBigIntPipe } from '@libs/api/pipes/parse-positive-int.pipe';
+
+@ApiTags('ChatMessage')
+@ApiInternalServerErrorBuilder()
```

Comment:
이번에 ApiKey가 추가돼서
```suggestion
@ApiInternalServerErrorBuilder()
@ApiSecurity('Api-Key')
```
이렇게 앞으로 컨트롤러 새로 생성할 때 데코레이터가 추가해주시면 감사하겠습니다

Reply:
넵 알겠습니다